openstack_controller/tests/terraform/main.tf

```diff
@@ -0,0 +1,78 @@
+variable "account_json" {
+  type = string
+}
+
+resource "random_shuffle" "az" {
+  input = ["europe-west2-a", "europe-west2-b", "europe-west2-c"]
+  result_count = 1
+}
+
+provider "google" {
+  credentials = var.account_json
+  project = "datadog-integrations-lab"
+  region = "europe-west2"
+  zone = "${random_shuffle.az.result[0]}"
+}
+
+resource "tls_private_key" "ssh-key" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "random_string" "suffix" {
+  length = 8
+  special = false
+  upper = false
+}
+
+resource "google_compute_instance" "devstack" {
+  name         = "devstack-${random_string.suffix.result}"
```

Comment:
You mean add another suffix?

Reply:
I agree that another suffix (or a prefix) is a good idea to know if something can be cleaned or not. However, I don't know the `.tf` file is the best place to do it since it can easily be forgotten. Is there a way to add a suffix in a python file?

Reply:
The idea would be to know what belongs to whom and to know if things can be removed. We've talked about using different service accounts to identify the created resources. Not sure how that would work but that'll be fine with me.

Reply:
One way to do something more obvious would be to get the system user and pass it to terraform. That would create instances like `devstack-thomas.herve-xyz`. WDYT?

Reply:
Having the integration name in the vm name could be helpful too: `devstack-e2e-<integration_name>-<whoami>-xyz`

Another option is to attach label to VMs:

Reply:
I added `getpass.getuser()` in the name.